### PR TITLE
Day 2

### PR DESCRIPTION
**Pull request by daquinons**

### CODE REVIEW

File: friends/src/components/AddEditFriend/AddEditFriend.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+    setFormInput({
+      name: "",
+      age: "",
+      email: ""
+    });
+  };
+
+  const onChangeInput = event => {
+    event.persist();
```
Comment (daquinons) [inline friends/src/components/AddEditFriend/AddEditFriend.jsx:60]: This is it! @GianBarGian

File: friends/src/App.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      await axios.post(URL, friend);
+      await getFriends();
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const updateFriend = async friend => {
+    try {
+      console.log(`${URL}/${friend.id}`);
```
Comment (GianBarGian) [inline friends/src/App.js:36]: Remember to clean your code from pointless console logs.
Also good job with the dynamic path here.

File: friends/src/components/AddEditFriend/AddEditFriend.jsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  const { onAddFriend, editableFriend, onEditFriend, onCancelEdit } = props;
+  const [formInput, setFormInput] = useState({
+    name: "",
+    age: "",
+    email: ""
+  });
+
+  useEffect(() => {
+    setFormInput({
+      name: editableFriend ? editableFriend.name : "",
```
Comment (GianBarGian) [inline friends/src/components/AddEditFriend/AddEditFriend.jsx:32]: This is cool! Well done.

File: friends/src/components/AddEditFriend/AddEditFriend.jsx (hunk 1 above)
Comment (GianBarGian) [reply friends/src/components/AddEditFriend/AddEditFriend.jsx:60]: Answered in the other PR. :-)

**Review by GianBarGian — COMMENTED** [review]
Good job, your project looks amazing, I love the styling and the edit and delete functionality works flawlessly.

**Pull request merged**